packages/driver/src/multi-domain/commands.ts

```diff
@@ -0,0 +1,52 @@
+import type { $Cy } from '../cypress/cy'
```

Comment:
I'm not the best at naming, but I am wondering if there is a way to name this file so that also implies that is contains the `log` events as well in addition to `commands`

Reply:
Decided the best route was to go ahead and separate the command and log handling altogether. Made naming a lot easier.

Reply:
Addressed in [`17a248a`](https://github.com/cypress-io/cypress/pull/19825/commits/17a248adb5919e6594eb5999c9e07ebc93b5c864)